Sources/Sentry/SentryTracer+Private.h

```diff
@@ -0,0 +1,16 @@
+#import "SentryTracer.h"
+
+@interface
+SentryTracer ()
+
```

Comment:
m: The SentryTracer is already private and will never be public, I don't see a problem with these properties being part of the SentryTracer, and then we will have one less header.

Reply:
Yep, makes sense! Will do.